### PR TITLE
[O2B-455] add flp home button

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- add configurable link-component to other GUI's on navBar just before 'Home' tab
- link to ali-flp main page visible as button with text - 'ALI FLP'

Notable changes for developers:
- backend utilize environmental variable:
    - ALI_FLP_INDEX_URL

Changes made to the database:
- NA

### CODE REVIEW

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/components/NavBar/index.js [hunk 1]
```diff
@@ -49,6 +50,29 @@ const pageLink = (model, href, text) => absoluteFrontLink(
     },
 );
 
+/**
+ * Return link-button (opens new tab) with given href
+ * @param {Object} conf configuration of the component
+ * @param {String} [conf.aliFlpIndexUrl] ali flp index url
+ * @returns {vnode} returns tab
+ */
+const aliFlpReferenceTab = (conf) => absoluteFrontLink(
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/components/NavBar/index.js]: done

File: lib/public/components/NavBar/index.js [hunk 2]
```diff
@@ -49,6 +50,29 @@ const pageLink = (model, href, text) => absoluteFrontLink(
     },
 );
 
+/**
+ * Return link-button (opens new tab) with given href
+ * @param {Object} conf configuration of the component
+ * @param {String} [conf.aliFlpIndexUrl] ali flp index url
+ * @returns {vnode} returns tab
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/components/NavBar/index.js]: done

File: lib/public/components/NavBar/index.js [hunk 3]
```diff
@@ -49,6 +50,29 @@ const pageLink = (model, href, text) => absoluteFrontLink(
...
+        target: '_blank',
+    },
+);
+
+const btnGroupsDelimiter = [
+    h('.ph1', { style: {
+        'border-rigth': 'medium groove',
+    } }), h('.ph1', { style: {
+        'border-left': 'medium groove',
+    } }),
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/components/NavBar/index.js]: done

File: lib/public/components/NavBar/index.js [hunk 4]
```diff
@@ -49,6 +50,29 @@ const pageLink = (model, href, text) => absoluteFrontLink(
...
+ * Return link-button (opens new tab) with given href
+ * @param {Object} conf configuration of the component
+ * @param {String} [conf.aliFlpIndexUrl] ali flp index url
+ * @returns {vnode} returns tab
+ */
+const aliFlpReferenceTab = (conf) => absoluteFrontLink(
+    'ALI FLP',
+    conf.aliFlpIndexUrl,
+    {
+        class: 'btn btn-group-item',
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/components/NavBar/index.js]: done

File: lib/public/components/NavBar/index.js [hunk 5]
```diff
@@ -16,6 +16,7 @@ import { iconPerson, iconPlus } from '/js/src/icons.js';
 import { userPreferencesStore } from '../../utilities/userPreferencesStore.js';
 import { frontLink } from '../common/navigation/frontLink.js';
 import { absoluteFrontLink } from '../common/navigation/absoluteFrontLink.js';
+import { getAliFlpReferencesConfiguration } from '../../services/externalRouting/getFlpIndexLinks.js';
```
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/components/NavBar/index.js]: done

File: lib/public/components/NavBar/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (martinboulais) [reply lib/public/components/NavBar/index.js]: Not entierly. Scalar in lowercase

File: lib/public/components/NavBar/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (xsalonx) [reply lib/public/components/NavBar/index.js]: deleted

**Review by martinboulais — APPROVED** [review]

**Pull request merged**